pkgs/development/python-modules/pmdsky-debug-py/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+#This package is auto-generated. It could totally be possible to generate it from upstream, but seems unecessary
+buildPythonPackage rec {
+  pname = "pmdsky-debug-py";
+  version = "3.0.4";
+  # SkyTemple specifically require 3.0.4. This is used when patching the binary,
+  # and risk to be a bit problematic if using the latest version, given it doesn’t follow semver.
+
+  src = fetchFromGitHub {
+    owner = "SkyTemple";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-kbubOolFsPZko/LULPxoHCfofdefo6OtfuYO0xpjOCw=";
+  };
+
+  prePatch = "cd src";
+
+  format = "pyproject";
+
+  meta = with lib; {
+    description = "Autogenerated and statically check-able pmdsky-debug symbol definitions for Python.";
```

```suggestion
    description = "Autogenerated and statically check-able pmdsky-debug symbol definitions for Python";
```

See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes